### PR TITLE
changed navbar

### PR DESCRIPTION
**Pull request by rahilsmd**
changed the navbar design.

### CODE REVIEW

**Comment by ananyo2012** [comment]
@rahilsmd Can you post some screenshots of the navbar? So that we can go for some quick fixes regarding styles if needed?

File: festflow/static/css/index.css [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 		border:  2px solid rgba(255,255,255, 0.3);
 	}
-	50%  {	background: rgba(255,255,255, 0.2);
+	50%  {	background: rgba(50,144,148, 0.4);
 		border:  2px solid rgba(255,255,255, 0.5);
 	}
-	100% {	background: rgba(255,255,255, 0.1);
+	100% {	background: rgba(50,144,148, 0.2);
 		border:  2px solid rgba(255,255,255, 0.3);
 	}
```
Comment (ghoshbishakh) [inline festflow/static/css/index.css]: @rahilsmd reason behind this change?

File: festflow/static/css/index.css [hunk 2]
```diff
@@ -23,22 +23,26 @@
 }
 
 #register-button {
-	background: rgba(255,255,255, 0.1);
+	background: rgba(50,144,148, 0.1);
```
Comment (ghoshbishakh) [inline festflow/static/css/index.css]: why?

**Comment by ghoshbishakh** [comment]
@ananyo2012 I have hosted rahils branch. take a look http://ank2016-nitd.rhcloud.com/

**Comment by ghoshbishakh** [comment]
@rahilsmd This navbar will not do. Change it. It does not load the pages properly. It uses some ajax that is not working well. In mobile devices it is unusable. If you change a page then the url does not change. And in all pages except the home page there is no navbar.

File: festflow/templates/festflow/sponsors.html [hunk 3]
```diff
@@ -1,165 +0,0 @@
-<!DOCTYPE html>
```
Comment (ghoshbishakh) [inline festflow/templates/festflow/sponsors.html]: wow you removed the entire page?

**Comment by ghoshbishakh** [comment]
@rahilsmd and for the event page it jumps to another page that actually has no back link or navbar.

**Comment by ghoshbishakh** [comment]
@ananyo2012 @rahilsmd Do you think this PR should continue? or will it be better to make the changes again from where he started?

**Comment by ghoshbishakh** [comment]
@rahilsmd why are there four commits? you should have reset and done it from the beginning

File: festflow/static/css/completeprofile.css [hunk 4]
```diff
@@ -19,3 +19,4 @@
 	font-family: 'Baloo Paaji' ,cursive;
 }
 
+
```
Comment (ghoshbishakh) [inline festflow/static/css/completeprofile.css]: ? why?

File: festflow/static/css/contact.css [hunk 5]
```diff
@@ -181,4 +182,758 @@ figure.effect-duke:hover p {
 	}
 }
 
+/*!
```
Comment (ghoshbishakh) [inline festflow/static/css/contact.css]: @rahilsmd why have you pasted entire bootstrap here?

Comment (ghoshbishakh) [inline festflow/static/css/contact.css]: ok not entire but a lot of. You could have simply used bootstrap as a separate include.

**Comment by ghoshbishakh** [comment]
screenshot from 2016-10-05 11-18-10

**Comment by ghoshbishakh** [comment]
I can never digest so many changes for this simple navbar.

**Comment by ghoshbishakh** [comment]
@rahilsmd why did you remove the preloader?

**Comment by rahilsmd** [comment]
I didn't remove it, just forgot to add

**Comment by rahilsmd** [comment]
If this navbar is fine, then we can go forward and I have reduced the
number of css and js files.

**Comment by ghoshbishakh** [comment]
it is fine for me @rahilsmd  but you must do a proper cleanup of the code

**Comment by ghoshbishakh** [comment]
@rahilsmd use proper classnames properly include bootstrap where necessary without pasting portions of it everywhere. remove unnecessary js

**Comment by rahilsmd** [comment]
Waited for a review, Now as it is fine I'll go forward and clean the code

**Comment by ananyo2012** [comment]
@rahilsmd We are going with the new navbar design in #13. So I think this can be closed now.

**Comment by ghoshbishakh** [comment]
closing and moving to #13

**Pull request closed without merging**